test/evergreen.yml

```diff
@@ -1638,6 +1662,105 @@ tasks:
               cp -rf WT_TEST WT_TEST_$file
             done
 
+  - name: check-test-long-1
```

Comment:
Can we rename this to something meaningful

Reply:
Well it is a long test which is doing whole bunch of stuff and perhaps that's why they called it just test-check-long :) . If I am not mistaken this test is sort of a longer version of `make check all` test with more scenarios. I am open to suggestions on naming it differently.

Reply:
Maybe `long-test`?

Reply:
Now I see why you break the tests into 2 tasks. The tests included in this task is something about no ftruncate setting. Please note we have an existing task `linux-no-ftruncate` which is scheduled to run once per day.

Reply:
I have to check what is the difference between setting ac_cv_func_ftruncate=no and what is being done in compile wiredtiger no linux ftruncate to confirm if we are just doubling the effort.